README.md

```diff
@@ -0,0 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Title
+
+To Do list: add & remove
+
+> This is a simple HTML list of To Do tasks.
```

Comment:
- [ ] [OPTIONAL] I think it would be better to add more description about your project to make it more understandable :wink: